tensorboard/webapp/persistent_settings/_redux/persistent_settings_actions.ts

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {createAction, props} from '@ngrx/store';
+
+import {PersistableSettings} from '../_data_source/types';
+
+/** @typehack */ import * as _typeHackModels from '@ngrx/store/src/models';
+
+/**
+ * Describes settings loaded from a global settings storage. Dispatched once
+ * when the application bootstraps.
+ */
+export const globalSettingsLoaded = createAction(
+  '[Persistent Settings] Global Settings Loaded',
```

Comment:
I'm confused.  I thought this pr is not for any global settings (across experiments) but for per experiment?

Reply:
It is exactly the opposite.